keylime/tenant.py

```diff
@@ -1249,27 +1238,21 @@ def do_add_allowlist(self, args):
         self.process_allowlist(args)
         data = {"tpm_policy": json.dumps(self.tpm_policy), "ima_policy_bundle": json.dumps(self.allowlist)}
         body = json.dumps(data)
-        cv_client = RequestsClient(self.verifier_base_url, self.tls_cv_enabled, ignore_hostname=True)
+        cv_client = RequestsClient(self.verifier_base_url, True, tls_context=self.tls_context, ignore_hostname=True)
```

Comment:
I feel a bit annoyed by this "True".  I suggest to make RequestsClient constructor to set `tls_enabled=True` and drop the parameter during the call?

Reply:
Yes, when I was writing the code I was a bit annoyed by this, but I thought the goal was to make it explicit that the hostname would be ignored when verifying certificates.

I'll change the code.